src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
```

Comment:
I think you can just keep the indentation of these "separator line" (?), or extract them into a separate function and call the function instead.

Reply:
I think this is a neat idea - especially if you want to modularise your responses later down the line.